type-morph/main.ts

```
@@ -171,6 +183,45 @@ function parseImportLine(line: string, language: string): { module: string; item
+    case 'java':
+    case 'kotlin':
+      const langMatch = line.match(/^import\s+([^;]+);$/);
+      if (langMatch) {
+        const fullPath = langMatch[1];
+        const parts = fullPath.split('.');
+        const className = parts[parts.length - 1];
+        return { module: fullPath, items: [className] };
+      }
+      break;
+    case 'go':
+      const goSingleMatch = line.match(/^import\s+"([^"]+)"$/);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)